onnxruntime/core/session/custom_ops.cc

```diff
@@ -66,6 +67,42 @@ ORT_API_STATUS_IMPL(OrtApis::KernelInfoGetAttribute_string, _In_ const OrtKernel
   return onnxruntime::ToOrtStatus(status);
 }
 
+template <typename T>
+static Status CopyNonStringDataFromVectorToMemory(const std::vector<T>& values, T* out, size_t* size) {
+  if (!std::is_fundamental<T>::value)
```

Comment:
Can we just have CopyDataFromVectorToMemory with an enable_if to limit to primitive types?

Reply:
Done